.github/workflows/lock.yaml

```diff
@@ -0,0 +1,24 @@
+name: 'Lock Threads'
+
+on:
+  schedule:
+    - cron: '0 * * * *'
```

Comment:
Does this cron run daily?

Reply:
This will run every hour since only 50 issues and pull requests will be handled at once. After that's processed, the schedule can be reduced.